addons/event_sale/models/event.py

```diff
@@ -60,6 +73,28 @@ def _is_event_registrable(self):
         else:
             return False
 
+    @api.depends('event_ticket_ids.beginning')
+    def _compute_start_sale_date(self):
+        for event in self:
+            event.start_sale_date = min(ticket.beginning for ticket in event.event_ticket_ids)
+
+    @api.depends('sale_order_lines_ids')
+    def _compute_sales_total_price(self):
+        for event in self:
+            event.sales_total_price = sum([
+                event.currency_id._convert(
```

Comment:
Side note: could you have lines beloging to another company ?

Reply:
@tde-banana-odoo not sure, but just in case, I can use the company of the sale order line